### PR TITLE
Removes hit and run from runner

### PR DESCRIPTION
**Pull request by Kuro-ai**
## About The Pull Request

Removes hit and run from runner

## Why It's Good For The Game

Hit and run with bump melee is broken

## Changelog
:cl:
balance: removes hit and run from runner
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm [hunk 1]
```diff
@@ -25,40 +24,7 @@
...
-	hit_and_run = 0 //reset the hit and run bonus
-	return COMSIG_XENOMORPH_BONUS_APPLIED // dont stack our attack with bonus_dam
-
-/mob/living/carbon/xenomorph/runner/handle_status_effects()
-	if(hit_and_run)
-		var/last_move = last_move_intent - 10
-		if(last_move && last_move < world.time - 5) //If we haven't moved in the last 500 ms, we lose our bonus
-			hit_and_run = 0
-	return ..()
+		COMSIG_XENOMORPH_ATTACK_LIVING))
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/xenomorph/castes/runner/runner.dm]: This is doing nothing.

**Comment by AdamElTablawy** [comment]
back to runners being shit again, i guess.

**Comment by jroinc** [comment]
+1. Bumpslash allows them to do consistent damage now, so this served it's purpose and isn't needed anymore.

**Comment by spookydonut** [comment]
> You currently have a negative Fix/Feature pull request delta of -2. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-4). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**